l5-operator/controllers/bestie.go

```diff
@@ -26,11 +26,13 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-const bestiePort = 8080
+const bestiePort = 8000
 const databaseServiceName = "mysql"
```

Comment:
can be removed

Reply:
Good call. I removed all references that pertain to MySQL.